spec/mailers/notifier_spec.rb

```diff
@@ -8,3 +8,45 @@
     expect(email.body).to include "service #{service_1.url}"
   end
 end
+
+describe 'ServiceChangeNotifier' do
+  include ServiceChangeNotifier
+  describe 'notify_services_of_changes' do
+    before(:each) do
```

Comment:
Since we figured out what was going on with `use_transactional_fixtures`, it'd be good to convert this back to `let` blocks.